lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -211,6 +233,39 @@ def test__has_access_descriptor_when_not_in_preview_mode(self):
         # Start date in the future.
         mock_unit.start = datetime.datetime.now(pytz.utc) + datetime.timedelta(days=1)  # release date in the future
         self.verify_access(mock_unit, False)
+        self.verify_error_type(mock_unit, access_response.StartDateError)
+
+    @patch.dict('django.conf.settings.FEATURES', {'DISABLE_START_DATES': False})
+    def test__has_access_descriptor_start_message(self):
```

Comment:
Is there a test that verifies Beta Users and Staff can bypass start date checks?